my_list.rb

```diff
@@ -0,0 +1,13 @@
+require './my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each()
+    @list.length.times { |i| yield @list[i] }
+  end
```

Comment:
- [ ] Kindly, make sure that you implemented an #each method correctly that yields successive members of all @list.
you can have a look on this implementation below:
```Ruby
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```

Reply:
- We changed the each function.